HHS/Head-Start-TTADP
# [TTAHUB-258] Visual fixes for dashboard and handle widget api errors

## Description of change
Visual fixes for the dashboard
- Align date label with heading on the Total Hours/TTA graph
- Fix the height of the reason list to always fill it's container
- Improve margins & paddings on date labels as the screen size shrinks
- Re-appear the disappearing tooltips

Reliability/performance fixes
- Fix lack of error handling for the Widget API. 
- Handle the specific error case in the Total hours grant that was causing an error

## How to test
View the /dashboard and verify the above cosmetic changes are in place
Verify that you can no longer obtain the "Can't load widget" errors.

## Issue(s)
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-258

## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
